core/src/main/scala/chisel3/VerificationStatement.scala

```diff
@@ -64,7 +64,7 @@ object assert extends VerifPrintMacrosDoc {
     * @param cond condition, assertion fires (simulation fails) on a rising clock edge when false and reset is not asserted
     * @param message optional chisel Printable type message
     *
-    * @note See [[printf.apply(fmt:Printable)]] for documentation on printf using Printables
+    * @note See `printf.apply(fmt:Printable)` for documentation on printf using Printables
```

Comment:
```suggestion
    * @note See [[printf.apply(pable:chisel3\.Printable)*]] for documentation on printf using Printables
```
I don't love using backticks for a specific function, so I figured out how to make it work, it seems you need no spaces, but correct argument name, and definitely need a `*` at the end. Also need to use `\.` if it seems to want a package included in the type of the argument.